Mac-App/Mac-App/AppDelegate.swift

```diff
@@ -13,29 +13,36 @@ import SwiftUI
 
 @NSApplicationMain
 final class AppDelegate: NSObject, NSApplicationDelegate {
-    private var window: NSWindow!
-    private var backendSubscription: BackendSubscription!
-    private let macoOSMenu = macOSMenu()
-
-    func applicationDidFinishLaunching(_: Notification) {
-        configureEnviromentValues()
-
-        window = NSWindow(
+    // MARK: - Private API
+    private lazy var window: NSWindow = {
```

Comment:
Does the window need to be initialized lazily as well?

Reply:
Hmm good point! So you're suggesting to insatiate the window right away?

Reply:
Yes, I haven't tested it but it looks like it would be fine 🤔  @Thurman1776-

Reply:
Ok, will give it a try!

Reply:
https://github.com/Thurman1776-/Uncomplect-us/pull/26/commits/b76f8f9362ca9f72bfcebe5e1fde4bb2b258d252